cmd/tdf-decrypt.go

```diff
@@ -115,6 +124,18 @@ func init() {
 		decryptCmd.GetDocFlag("no-verify-assertions").DefaultAsBool(),
 		decryptCmd.GetDocFlag("no-verify-assertions").Description,
 	)
+	decryptCmd.Flags().Bool(
+		decryptCmd.GetDocFlag("ignore-kas-allowlist").Name,
+		decryptCmd.GetDocFlag("ignore-kas-allowlist").DefaultAsBool(),
+		decryptCmd.GetDocFlag("ignore-kas-allowlist").Description,
+	)
+	decryptCmd.Flags().StringSliceVarP(
+		&kasAllowList,
+		decryptCmd.GetDocFlag("kas-allowlist").Name,
+		decryptCmd.GetDocFlag("kas-allowlist").Shorthand,
+		nil,
+		decryptCmd.GetDocFlag("kas-allowlist").Description,
+	)
```

Comment:
Could we reduce this to be just `kas-allowlist` and have a special variable for `*` to allow anything rather than another flag?

Reply:
good call, updated

Reply:
Otherwise we'd have to determine how we handle

```
otdfctl --ignore-kas-allowlist --kas-allowlist "https://kas1.example.com" --kas-allowlist "https://kas2.example.com"
```

While this seems pretty s

```
otdfctl --kas-allowlist "*" --kas-allowlist "https://kas.example.com"
```